sensor_msgs/Image image
string position